encoder-brave/src/main/java/zipkin2/reporter/otel/brave/BraveScope.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.reporter.otel.brave;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.util.Properties;
+
+/**
+ * Define InstrumentationScope for encoder-brave
+ */
+final class BraveScope {
+  static final String name;
+
+  static final String version;
+
+  static {
+    try (InputStream stream = BraveScope.class.getClassLoader().getResourceAsStream("scope.properties")) {
+      if (stream != null) {
+        Properties props = new Properties();
+        props.load(stream);
+        name = props.getProperty("name");
+        version = props.getProperty("version");
+      }
+      else {
+        name = "unknown";
+        version = "unknown";
+      }
+    }
+    catch (IOException e) {
+      throw new UncheckedIOException(e);
+    }
+  }
+
+  public static String getName() {
```

Comment:
Since the class isn't public guess we can remove from the methods (I guess this changed to a static class to handle resource loading)

Or since the fields are exposed anyways, maybe can remove the methods entirely